docs/Installation/System-Requirements.md

```diff
@@ -0,0 +1,29 @@
+title: System Requirements
+description: System requirements to sync and run Pantheon 
+<!--- END of page meta data -->
+
+# System Requirements 
+
+The system requirements for Pantheon depend on a number of factors: 
+
+* Network - are you connecting to a public Ethereum network (eg, mainnet) or a private network 
+* Network size if connecting to a private network 
+* Number and complexity of queries being performed by node 
+
+## Determining System Requirements  
+
+To determine system requirements, monitor CPU and disk space requirements using [Prometheus](https://docs.pantheon.pegasys.tech/en/stable/Monitoring/Monitoring-Performance/#monitor-node-performance-using-prometheus). 
+A sample [Grafana dashboard](https://grafana.com/grafana/dashboards/10273) is provided for Pantheon. 
+
+!!! tip
+    CPU requirements are highest when syncing to the network and reduce once the node is synchronized to the chain head. 
```

Comment:
It's not guaranteed that less CPU will be used when synchronised.

```suggestion
    CPU requirements are highest when syncing to the network and will typically reduce once the node is synchronized to the chain head. 
```

Reply:
Fixed